cyhy_commander/jobs/vulnscan.py

```diff
@@ -338,12 +343,17 @@ def main():
     LOGGER.info("Using ports file: %s", ports_file)
 
     # read in targets and ports
-    with open(targets_file, "r") as f:
+    with open(targets_file) as f:
         targets = f.readlines()
 
-    with open(ports_file, "r") as f:
+    with open(ports_file) as f:
         ports = f.readline().strip()
 
+    # Before beginning work that involves the Nessus Controller we will sleep for a
+    # random amount of time (range of [0, MAX_SLEEP_SEC+1) seconds) to help prevent
+    # Nessus from getting overloaded by multiple jobs starting at the same time.
+    time.sleep(secrets.randbelow(MAX_SLEEP_SEC + 1))
```

Comment:
It doesn't matter much, but why `MAX_SLEEP_SEC + 1` here?  This makes the maximum sleep 30.999... seconds instead of 29.999... seconds as I would expect.

Reply:
`secrets.randbelow(31)` will return a random integer between 0 and 30.  From https://docs.python.org/3/library/secrets.html#secrets.randbelow and also mentioned in the PR description by @mcdonnnj:
```
Return a random int in the range [0, n).
```
Did you maybe misread that as `[0, n]`?

Reply:
Ah, it returns an integer.  That's what I was missing.